Makefile

```diff
@@ -83,7 +83,7 @@ race_condition_run: race_condition_binary
 	./prometheus.race $(ARGUMENTS)
 
 run: binary
```

Comment:
Let's get rid of the `run` target completely. It was useful in the times where you had to set a lot of env variables to point to all the shared libraries (LevelDB and deps), but is useless now, especially since logging is the default now.